protocol/x/prices/keeper/market.go

```diff
@@ -74,17 +74,22 @@ func (k Keeper) CreateMarket(
 	return marketParam, nil
 }
 
-// IsRecentlyAdded returns true if the market was added recently. Since it takes a few seconds for
-// index prices to populate, we would not consider missing index prices for a recently added market
-// to be an error.
-func (k Keeper) IsRecentlyAdded(marketId uint32) bool {
+// IsRecentlyAvailable returns true if the market was recently made available to the pricefeed daemon. A market is
```

Comment:
This method controls when missing index prices are escalated to errors in the logs. When a node starts with a delayed genesis time, the market is created on node start, but prices are still not available until the daemon starts, so this logic was updated to use block height as a proxy for daemon warmup period.

Reply:
nit: Can you include the reasoning about the block height being used as a proxy into the comment?